### ISSUE TITLE
Listing fonts crashes the app

### ISSUE THREAD

**Issue by tleylan** [giuspen/cherrytree#1763]
I've tried this several times using ver: 0.99.39 (I installed it yesterday) on Windows 10.

File
Preferences
Fonts
in rich text it says "None 20"
The Pick a Font dialog lists only fonts that start with "S" so Sans Normal to Sitka <something>. It crashes as I get to Sitka.

I can select Sans Normal and I can adjust the font size without issue but what about all the other fonts? And of course it shouldn't crash.  Oh I just tried scrolling upwards and there are some other fonts but it eventually crashes as well.

Tom

**Comment by giuspen** [comment]
Is it crashing as you select a font or just scrolling?

**Comment by tleylan** [comment]
Definitely while scrolling (and I just did it again as a test). I was able to select one of the first few fonts offered I don't know if that would have worked further down the list. This time I got the cherrytree.exe has stopped working dialog. The Sitka Text Bold Oblique choice is visible I was scrolling past that one.  Ah, I tried to start the debugger and I got some additional info.

_Unhandled exception at 0x00007FFB07F6760B (libpango-1.0-0.dll) in cherrytree.exe: 0xC0000005: Access violation reading location 0x0000000000000000_.

**Comment by giuspen** [comment]
It seems a bug in the current library msys2 build

**Comment by tleylan** [comment]
Hi there. That's unfortunate when you don't have control of the other libraries. Am I correct in assuming that you will release an update when/as this library is updated? I don't build the project from source. Thanks.

**Comment by giuspen** [comment]
Yes @tleylan I'm constantly upgrading the libraries to the latest stable version. The libraries come from https://www.msys2.org/ Hopefully will be ok in 0.99.40 if not I'll spend more time in trying to reproduce and possibly to work around the issue

**Comment by tleylan** [comment]
I figured as much, thanks again. If I can be of help let me know.

**Comment by joetbl** [comment]
Hi there, the problem is also in the version 0.99.40. If you click to change the font, the application crashed. (app/settings/fonts)

**Comment by giuspen** [comment]
In the preferences Dialog, tab Miscellaneous, it is possible to enable the Debug Log.
Could please you do it, then reproduce the issue and attach the log here so I will have a look and see if it helps?

**Comment by joetbl** [comment]
Hi, here ist the output from the debug log.
_[2021-08-13 14:31:45.212] [gtk] [critical] Class::register_derived_type(): base_query.type_name is NULL.
[2021-08-13 14:31:45.590] [   ] [debug] C:\Users\***********\AppData\Local\cherrytree\config.cfg parsed
[2021-08-13 14:31:45.799] [   ] [debug] autosave is started
[2021-08-13 14:32:00.484] [   ] [debug] Node name: ******
[2021-08-13 14:32:00.910] [   ] [debug] fs::download_file: start downloading https://www.giuspen.com/software/version_cherrytree
[2021-08-13 14:32:03.917] [   ] [error] fs::download_file: curl_easy_perform() failed, Timeout was reached
[2021-08-13 14:32:05.939] [   ] [debug] shift images in MenuBar/context menu_

After open the font windows it shows only the window title. Then Cherrytree crashed an close automatic both windows.
